src/injector.c

```diff
@@ -128,13 +128,20 @@ static inline void print_help(void)
             "\t -d <domain ID or name>    The domain's ID or name\n"
             "\t -i <injection pid>        The PID(WIN) | TGID(LINUX) of the process to hijack for injection\n"
             "\t -e <inject_file>          The executable to start with injection\n"
+            "\t                             for -m readfile/writefile: the guest path of the file to be read/written\n"
```

Comment:
These should either be separate args or both descriptions should be the same

Reply:
Sorry, could you explain?

Reply:
```suggestion
            "\t -e <inject_file>            for -? injectcode: executable to start with injection\n"
            "\t                             for -m readfile/writefile: the guest path of the file to be read/written\n"
```
smth like this